src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
Each object has a parent, but GFF3 also allows a derived_from tag, kinda
like a secondary ancestor link.

That means that although Gene1 is parent to Transcript1 and Transcript1 is
parent to Exon1, you are free to have Exon1 derived from another gene.

On Wed, Feb 4, 2015 at 6:17 AM, Mark Diekhans <notifications@github.com>
wrote:

> In src/main/resources/avro/sequenceAnnotations.avdl
> <https://github.com/diekhans/ga4gh-schemas-genome-ann/pull/16#discussion_r24064652>
> :
>
> >      */
> > -    array<string> childrenIds;
> > +    string parentId;
>
> GFF3 does use multiple parent ids for trans-splicing representations.
> Ideas?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diekhans/ga4gh-schemas-genome-ann/pull/16/files#r24064652>
> .
>